xml/slert_shielding_model.xml

```diff
@@ -4,83 +4,103 @@
+     are run in the <systemitem>user</systemitem> &cpuset;.
+    </para>
+   </formalpara>
+  </listitem>
+ </itemizedlist>
+
  <para>
   The <command>shield</command> subcommand manages all of these &cpuset;s and
-  lets you define the CPUs and Memory Nodes that are in the
-  <literal>shielded</literal> and <literal>unshielded</literal> sets. The
-  subcommand automatically moves all movable tasks on the system into the
+  lets you define the CPUs and memory nodes that are in the
+  <literal>shielded</literal> and <literal>unshielded</literal> sets. 
+  The subcommand automatically moves all movable tasks on the system into the
   <literal>unshielded</literal> &cpuset; on shield activation, and back into
-  the <literal>root</literal> &cpuset; on shield tear down. The subcommand
+  the <systemitem>root</systemitem> &cpuset; on shield tear down. The subcommand
   lets you move tasks into and out of the shield. Additionally, you can move
   special tasks (kernel threads) which normally run in the
-  <literal>root</literal> &cpuset; into the <literal>unshielded </literal> set
-  so that your shield will have even less disturbance.
+  <systemitem>root</systemitem> &cpuset; into the <literal>unshielded</literal>
+  set. This makes your shield have even less disturbance.
  </para>
  <para>
-  The shield subcommand abstracts the management of these &cpuset;s away from
-  you and provides options that drive how the shield is set up, which tasks
-  are to be shielded and which tasks are not, and status of the shield. In
+  The <command>shield</command> subcommand abstracts the management of these &cpuset;s away from
+  you. It provides options that drive how the shield is set up, which tasks
+  are to be shielded or not, and the status of the shield. In
   fact, you need not be bothered with the naming of the required &cpuset;s or
   even where the &cpuset; file system is mounted. <command>cset</command> and
   the <command>shield</command> subcommand takes care of all that.
  </para>
  <para>
-  If you need to define more &cpuset;s for your application, then it is likely
-  that this simple shielding is not a rich enough model for you. In this
+  If you need to define more &cpuset;s for your application, it is likely
+  that this simple shielding is not rich enough for you. In this
   case, you should transition to using the <command>set</command> and
   <command>proc</command> subcommands described in
   <xref linkend="cha.shielding.cpuset"/>.
  </para>
+
  <sect1 xml:id="sec.shielding.model.ex">
   <title>A Simple Shielding Example</title>
-
-  <para>
-   Assume that we have a 4-way machine that is not NUMA. This means there
-   are 4 CPUs at our disposal and there is only one Memory Node available.
-   On such machines, we do not need to specify any memory node parameters to
-   cset, it sets up the only available memory node by default.
-  </para>
+  <para> Assume that we have a 4-core machine that is not NUMA. This means
```

Comment:
Not completely sure about the question.  Looking at SLERT documentation I can see proper use of the term "NUMA".  It is only "non-NUMA" which is evil as it expands into "non-Non Uniform Memory Access" which is a double negation. So no more changes needed here from my POV.